storage/inchi/src/main/java/org/openscience/cdk/inchi/InChIOutputToStructure.java

```diff
@@ -0,0 +1,355 @@
+package org.openscience.cdk.inchi;
```

Comment:
Please add the missing copyright header.

Reply:
Please make sure to also copy the original copyright info from the file this was copied from, `Copyright (C) 2006-2007  Sam Adams <sea36@users.sf.net>`, I believe.